source/cellular_setup.c

```diff
@@ -189,6 +189,19 @@ bool setupCellular( void )
         }
     }
 
+    #ifdef CELLULAR_MODEM_NO_EPS_NETWORK
+        if( cellularStatus == CELLULAR_SUCCESS )
+        {
+            /*Quectel GSM Modules require starting the TCPIP task*/
+            cellularStatus = Cellular_StartTCPIP( CellularHandle );
```

Comment:
Cellular_StartTCPIP is qgsm modem specific APIs.
Suggest to make a copy to "source/cellular/qgsm/cellular_setup_qgsm.c".